### PR TITLE
Feature/add more components

### LINKED ISSUE #120: Add basic components to the ThemeProvider Story
It would be very useful to see how do the normal components `Button, Switch, Input etc.` look with the theme applied.

We should group them based on their category so it looks visually appealing and structured.

### PR DESCRIPTION
**Pull request by rosudhi**
## This PR Closes #120 

- [x] Add more basic components

We can see clearly now which components needs to be improved in terms of theming

### CODE REVIEW

**Comment by rihr** [comment]
Should work now. I also changed the lint step to not cause the rest of the pipeline to be cancelled when there are lint errors - but only in PRs, so if you merge with lint errors, it won't deploy

**Comment by rosudhi** [comment]
super thanks!

File: packages/react-components-lab/src/components/ThemeProvider/ComponentsMui/ComponentItem.tsx [hunk 1]
```diff
@@ -73,14 +73,15 @@ const ComponentItem: React.FC<ComponentItemProps> = forwardRef<
...
+                  code={`import { ${item.title.replace(
+                    ' ',
+                    ''
+                  )} } from '@material-ui/core'\n\n${beautifyCode(c)}`}
                 />
               </Collapse>
             </>
           )}
-          {isLastItem && <div style={{ marginBottom: 320 }} />}
+          {/* {isLastItem && <div style={{ marginBottom: 320 }} />} */}
```
Comment (rihr) [inline packages/react-components-lab/src/components/ThemeProvider/ComponentsMui/ComponentItem.tsx:84]: There's a lint warning above about isLastItem prop being unused, considered removing both references or commenting the prop?

**Review by rihr — APPROVED** [review]

File: packages/react-components-lab/src/components/ThemeProvider/ComponentsMui/ComponentItem.tsx (hunk 1 above)
Comment (rosudhi) [reply packages/react-components-lab/src/components/ThemeProvider/ComponentsMui/ComponentItem.tsx:84]: yes, i will just remove the props, i think that's not necessary anymore

**Pull request merged**